### PR TITLE
Refactor config storage to use modules

### PR DESCRIPTION
**Pull request by chmelevskij**
Use esm module to handle interaction with config storage.

This PR replaces all the usages in code which is module based. There are few parts of the system which are not yet esm so those parts are still using old global scope approach.

### CODE REVIEW

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
